python/lessons_help_wanted.py

```diff
@@ -0,0 +1,182 @@
+# Import all the things
+import requests
+import os 
+from datetime import datetime
+import time
+import traceback
+import json
+
+# Set global vars
+GH_ORG_DC = "datacarpentry"
+GH_ORG_SWC = "swcarpentry"
+GH_ORG_LC = "librarycarpentry"
+GH_ORG_LAB = "carpentries-lab"
+GH_ORG_INC = "carpentries-incubator"
+ALL_ORGS = [GH_ORG_DC, GH_ORG_LC, GH_ORG_SWC, 'notanorg', GH_ORG_LAB, GH_ORG_INC, ]
+REPO_SEARCH_BASE_URL = "https://api.github.com/search/repositories"
+
+topics = "stable,helpwanted-list"
+labels = "\"help wanted\",\"good first issue\""
+
+# Authentication
+GH_TOKEN = os.environ['GITHUB_PAT_HELPWANTED']
+
+headers = {
+    "Authorization": f"token {GH_TOKEN}",
+    "Accept": "application/vnd.github+json"
+}
+
+
+def get_repos_by_topic(org, repo_topics):
+    """
+    Function takes:
+    * GH organization name as a string (eg "swcarpentry" or "carpentries")
+    * GH repo topics to search for as a csv string. 
+    Queries for all GH repos in that organization that have the topic "stable".
+    Returns list of dicts with repo and org data (to be used to gather help-wanted issues in that repo).
+    """
+
+    print(f"Searching for repos in {org} with topics: {topics}")
+    # Get GH organization name and url
+    org_url = f"https://api.github.com/orgs/{org}"
+    gh_org_req = requests.get(org_url, headers=headers)
+    if gh_org_req.status_code != 200:
+        print(f"Failed to retrieve repos by organization: {gh_org_req.status_code}, {gh_org_req.json()}")
+        return gh_org_req.status_code
+
+    gh_org = gh_org_req.json()
+    org_full_name = gh_org['name']
+    org_url = gh_org['html_url']
+
+    # Get all repos in that organization the topic "stable"
+    # Additional topics can be added separated by commas and use OR logic.
+    params = {
+        "q": f"org:{org} topic:{repo_topics}",
+    }
+
+    repos_with_topic_req = requests.get(REPO_SEARCH_BASE_URL, headers=headers, params=params)
+
+    if repos_with_topic_req.status_code != 200:
+        print(f"Failed to retreive repos: {repos_with_topic_req.status_code}, {repos_with_topic_req.json()}")
+        return repos_with_topic_req.status_code
```

Comment:
You could replace these requests.get calls with a reusable function that would make the code more maintainable:

`do_get(BASE_URL, etc)`